Solitary-Snipe Fixes For Jetstream

amercer1 opened the pull request:
Highlight Reel:

- [x] Update atmosphere-ansible to the latest from [upstream repo](https://github.com/iPlantCollaborativeOpenSource/atmosphere-ansible)
- [x] Mount volumes to use ext4 by default
- [x] DHCP exit hook to *not* use external IP resolution services by default
- [x] Remove ntp configuration as a step 
- [x] Variable-ize fail2ban and adjust `max_retry`, `findtime`, and `bantime` defaults

Tharon-C committed:
Add umount and mount volume role

c-mart committed:
Merge pull request #1 from iPlantCollaborativeOpenSource/master

merge changes from upstream

amercer1 committed:
Update vnc commands for centos 5

nfaction committed:
Add tasks to remove irods and icommands
	modified:   atmo-irods/tasks/main.yml

amercer1 committed:
Add tasks to remove any dead links
	modified:   tasks/main.yml

Commit:
Installing new iCommands 4.1.9 for CentOS 6+7 and all Ubuntu

c-mart committed:
Merge pull request #48 from amercer1/master

Remove old installations of icommands and irods

Commit:
Merging Andre's changes with my own, removing old iCommands and installing new iCommands

Commit:
now getting installation packages from Amazon S3

steve-gregory committed:
Add python interpreter call for centos 5
	modified:   tasks/main.yml

amercer1 committed:
Add fall back os variable files
	modified:   tasks/main.yml
	renamed:    vars/default.yml -> vars/CentOS.yml
	new file:   vars/Ubuntu.yml

amercer1 committed:
Merge pull request #49 from iPlantCollaborativeOpenSource/irods-4.1.9-install

Irods 4.1.9 installation

steve-gregory committed:
Add tasks to set python interpreter for centos 5
	modified:   atmo-irods/tasks/main.yml

Commit:
Adding idrop role back in

Commit:
Revert "Adding idrop role back in" - accidentally committed to master

This reverts commit f9c560f163f97982b2b74c8a25e4639f731ce878.

amercer1 committed:
Add popular files to .gitignore
	modified:   ../.gitignore

Commit:
instructions to use new role template

Commit:
instructions to use new role template

amercer1 committed:
Add playbook/roles to mount and unmount devices
	new file:   ../playbooks/utils/mount_volume.yml
	new file:   ../playbooks/utils/unmount_volume.yml
	modified:   mount-volume/tasks/main.yml
	deleted:    umount-volume/tasks/main.yml
	new file:   unmount-volume/tasks/main.yml

amercer1 committed:
Add steps to reduce required parameters
	modified:   unmount-volume/tasks/main.yml

amercer1 committed:
Add loop feature to allow a device to be unmounted from multiple mountpoints
	modified:   unmount-volume/tasks/main.yml

amercer1 committed:
Fix name header
	modified:   unmount-volume/tasks/main.yml

amercer1 committed:
Add unmount-volume/tasks/main.yml
	modified:   unmount-volume/tasks/main.yml

amercer1 committed:
Add conditional checks for required variables
	modified:   playbooks/utils/mount_volume.yml
	modified:   playbooks/utils/unmount_volume.yml

steve-gregory committed:
Merge pull request #52 from iPlantCollaborativeOpenSource/mount-and-umount

Mount and umount

amercer1 committed:
Merge branch 'master' of https://github.com/iPlantCollaborativeOpenSource/atmosphere-ansible

amercer1 committed:
Add check to see if gateone/shell can talk to the vm
	modified:   roles/atmo-check-novnc/defaults/main.yml
	modified:   roles/atmo-check-novnc/tasks/main.yml

steve-gregory committed:
Merge pull request #53 from iPlantCollaborativeOpenSource/gateone-connection

Check novnc proxy network from the gateone connection

steve-gregory committed:
Merge pull request #51 from c-mart/master

instructions to use new Ansible role template

amercer1 committed:
Merge remote-tracking branch 'upstream/master'

amercer1 committed:
Change default from ext3 to ext4
	modified:   ansible/roles/mount-volume/defaults/main.yml

amercer1 committed:
Remove atmo-irods
	deleted:    ansible/roles/atmo-irods/tasks/main.yml
	deleted:    ansible/roles/atmo-irods/vars/CentOS-5.yml
	deleted:    ansible/roles/atmo-irods/vars/CentOS-6.yml
	deleted:    ansible/roles/atmo-irods/vars/CentOS-7.yml
	deleted:    ansible/roles/atmo-irods/vars/CentOS.yml
	deleted:    ansible/roles/atmo-irods/vars/Ubuntu-12.yml
	deleted:    ansible/roles/atmo-irods/vars/Ubuntu-14.yml
	deleted:    ansible/roles/atmo-irods/vars/Ubuntu-16.yml
	deleted:    ansible/roles/atmo-irods/vars/Ubuntu.yml

steve-gregory commented on ansible/roles/atmo-novnc/vars/CentOS-5.yml:
```diff
@@ -51,11 +51,11 @@ REQUIRED:
+  #- "/bin/su {{ ATMOUSERNAME }} -c '/usr/bin/vncserver -localhost :{{ NOVNC.vnc_session }} -vncconfigfile {{ NOVNC.config_file }}'"
   # To disable NoVNC web when entering prod, remove "--web /usr/share/novnc/" from the command below
-  - "/opt/kanaka-noVNC-8b0a0f6/utils/launch.sh --listen {{ NOVNC.socket_port }} --vnc localhost:{{ NOVNC.vnc_port }} &"
+  #- "/opt/kanaka-noVNC-8b0a0f6/utils/launch.sh --listen {{ NOVNC.socket_port }} --vnc localhost:{{ NOVNC.vnc_port }} &"
```
Where are these ports defaults listed?

amercer1 replied:
In the defaults [file](https://github.com/jetstream-cloud/atmosphere-ansible/blob/integrate-mount-roles/ansible/roles/atmo-novnc/defaults/main.yml).

steve-gregory replied:
Perfect!

steve-gregory reviewed (approved):
Looks good, in the future I would suggest bringing in upstream changes separate from making playbook changes.

amercer1 committed:
Jetstream does not want the NTP configuration templates installed.
	modified:   main.yml

amercer1 committed:
Update vars for fail2ban
	modified:   templates/Ubuntu-15.jail.local.j2
	modified:   templates/centos-jail.local.j2
	modified:   templates/centos7-jail.local.j2
	modified:   templates/ubuntu-jail.local.j2
	modified:   vars/main.yml

amercer1 committed:
Variablize mountpoints
	modified:   atmo-setup-mount-point/defaults/main.yml
	modified:   atmo-setup-mount-point/tasks/main.yml

amercer1 committed:
Variablize group in /etc/group
	modified:   defaults/main.yml
	modified:   tasks/main.yml

amercer1 committed:
Update dhcp role with template
	new file:   roles/atmo-dhcp/defaults/main.yml
	modified:   roles/atmo-dhcp/tasks/main.yml
	renamed:    roles/atmo-dhcp/files/hostname-exit-hook.sh -> roles/atmo-dhcp/templates/hostname-exit-hook.sh.j2

amercer1 committed:
Add code to remove deprication warnings
	modified:   roles/atmo-cleanup/tasks/main.yml
	modified:   roles/atmo-user-boot-scripts/tasks/main.yml

steve-gregory commented:
Adjusting title and description to properly reflect the new scope of the PR

steve-gregory commented:
:shipit: Thanks again @amercer1 !
